server/models/visits.js

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var q = require('q');
```

Comment:
See [previous comment](https://github.com/mozilla/chronicle/pull/224/files#r23794554) re: changing this to
```js
var Q = require('q');
```